### ISSUE TITLE
Idea: notify by email when adding a new vault member

### ISSUE THREAD

**Issue by pradel** [Trust-Machines/multisafe-ui#10]
In order to make the process easier, wdyt about adding a new field in this modal (optional) so I can enter the email of that person. An email will be sent with a link so he can be notified that he was added to a vault (with a direct link inside)?

![Screenshot 2022-09-13 at 11 48 57](https://user-images.githubusercontent.com/5749437/189870334-bcb56b08-7824-4b1e-a881-a124e76a409c.png)

**Comment by talhasch** [comment]
Good idea! It can be implemented using a combination of our backend (we are currently building) + smart contract. A smart contract level solution would be possible. Let's see options..
